### ISSUE TITLE
Bug with messages

### ISSUE THREAD

**Issue by AxerTwo** [EvModder/DropHeads#35]
still getting messages on chat
# How to announce beheadings, options are GLOBAL, LOCAL, DIRECT, or OFF
# To specify per entity, use: "entity_name: '<MODE>'"
behead-announcement:
  default: 'OFF'
  player: 'OFF'
#  wither_skeleton: 'OFF'

# Replaces death messages if player behead announcement is GLOBAL
behead-announcement-replaces-death-message: false

**Comment by AxerTwo** [comment]
![image](https://user-images.githubusercontent.com/83673862/117071750-c67bc380-ad2f-11eb-86cf-b4eff7850dc9.png)

**Comment by EvModder** [comment]
Hmm, I just tested and it works for me.
Can you verify what version you are using and share the config.yml on https://pastebin.com/ maybe?

**Comment by AxerTwo** [comment]
i'm using 
DropHeads v3.6.1 (I've updated today and still have the same problem, 2 days ago plugin works fine(3.05.2021)
https://pastebin.com/tpKDNUi3

**Comment by EvModder** [comment]
did you restart/reload the server (or reload the plugin) after making the config changes? I tried with your config and it still hides the messages for me

**Comment by AxerTwo** [comment]
I restarted the server and reloaded with command /reload

**Comment by AxerTwo** [comment]
This is my Plugins (24): AutoMessage, BossBarMessage*, ChatColor2, ColoredSigns*, CombatLogX, CraftserveAgent*, DeathMessages, DropHeads, EasySetSpawn*, Essentials, Harbor, Join-Leave*, LPC, LuckPerms, nLogin, ProtectionStones, RHLeafDecay, SkinsRestorer, SkQuery, Skript, TAB, TreeCapitator*, WorldEdit, WorldGuard

**Comment by EvModder** [comment]
Actually, I found out 1 other possible way this could be happening...
if you are vanished or invisible, or have '`dropheads.silentbehead`' permission, it automatically changes the announcement mode to "`DIRECT`", meaning it sends the behead message to the person who killed the mob (but nobody else).

This is probably the **REAL** source of the bug; it should only be changing the mode when the mode isn't already '`OFF`' I can go ahead and fix this right away and upload v3.6.2 :)

**Comment by AxerTwo** [comment]
Okay, I will try

**Comment by EvModder** [comment]
Marking as fixed because it should be working now, feel free to leave a comment if the issue is still happening for you (that will reopen the bug)

**Comment by AxerTwo** [comment]
![image](https://user-images.githubusercontent.com/83673862/118120233-d65e6c00-b3ef-11eb-8f8c-c934b33dea27.png)
Still have this bug (version from today 13.05.2021) I tried to disable some plugins and still getting this annoucements
My code (from newest version)
https://pastebin.com/4vKHSPWK

**Comment by EvModder** [comment]
Hmm okay, I'll look at it more. Do the messages get sent to just yourself or other players as well?
Also, a temporary fix would be to edit these fields in the config:
```message-beheaded: '&6${VICTIM}&r was decapitated'
message-beheaded-by-entity: '&6${VICTIM}&r was decapitated by &6${KILLER}&r'
message-beheaded-by-entity-with-item: '&6${VICTIM}&r was decapitated by &6${KILLER}&r'
message-beheaded-by-entity-with-item-named: '&6${KILLER}&r decapitated &6${VICTIM}&r using &7${ITEM}&r'```

changing them to this should also completely hide them:
```message-beheaded: ''
message-beheaded-by-entity: ''
message-beheaded-by-entity-with-item: ''
message-beheaded-by-entity-with-item-named: ''```

**Comment by EvModder** [comment]
Sorry for earlier, I marked it as fixed but the fix didn't actually get included until the v3.6.5 release (oops!)
It should be _properly_ fixed now.
